extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/config/build/OTelBuildConfig.java

```diff
@@ -37,6 +37,14 @@ public interface OTelBuildConfig {
     @WithDefault("true")
     boolean enabled();
 
+    /**
+     * Should we use simple span and log handling.
```

Comment:
Probably: Should we use simple processor for spans and log records. This will disable batch processing and the exporter will send telemetry data right away. This is recommended for serverless applications.

Reply:
OK, fixed / changed.